integration/local

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+IMAGE=ghcr.io/lichess-org/lila-docker:main
+
+cleanup_containers() {
+    docker rm --force lila > /dev/null 2>&1 || true
+    docker rm --force app > /dev/null 2>&1 || true
+    docker network rm lila-network > /dev/null 2>&1 || true
+}
+
+echo "Running integration tests"
+cleanup_containers
+
+docker network create lila-network
+docker run --name lila --network lila-network -d $IMAGE
+docker run --name app --network lila-network -v $(pwd):/app -w /app $IMAGE ./integration/run-tests.sh
```

Comment:
Sorry for the delay! Finally looking at this, testing work. I don't understand why we need 2 containers?

Reply:
Ah I see, one is running Lila, the other only the test, I was wondering since they both use the same image